.github/workflows/release-doc.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish release changelog
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build Changelog
+        id: pmm_release
+        run: |
+          git clone --depth 1 https://github.com/percona/pmm-doc doc
+          cd doc/docs/release-notes
+          version="${release_tag/refs\/tags\/v//}"
+          CHANGELOG=`cat ./$version.md`
+          echo "CHANGELOG<<EOF" >> $GITHUB_ENV
+          echo "$CHANGELOG" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+        env:
+          release_tag: ${{ github.ref }}
+      - name: Create Release
+        uses: softprops/action-gh-release@v0.1.14
```

Comment:
Please ask somebody with proper permissions to clone this action to percona namespace. We don't allow external action in our workflows.

Reply:
@atymchuk can we clone this release?

Reply:
@BupycHuk Done, you can try now

Reply:
And what it the idea?

Reply:
Adding softprops/action-gh-release to allowed github actions list :)

Reply:
But why? That creates security breach, no?

Reply:
Discussed with Alex. We will use pinned versions + constraints in repo.